zerver/lib/exceptions.py

```diff
@@ -171,6 +172,20 @@ def extra_headers(self) -> dict[str, Any]:
         return extra_headers_dict
 
 
+class StreamExistsError(JsonableError):
+    code = ErrorCode.STREAM_ALREADY_EXISTS
+    http_status_code = 409
+    data_fields = ["stream"]
+
+    def __init__(self, stream: str) -> None:
+        self.stream = stream
+
+    @staticmethod
+    @override
+    def msg_format() -> str:
+        return _("Channel '{stream}' already exists")
```

Comment:
This local variable might as well be called `channel_name` to save migration work later.